src/cljs/harja/views/ilmoitukset/tieliikenneilmoitukset.cljs

```diff
@@ -415,8 +416,11 @@
                               (boolean (valitut-ilmoitukset rivi))]]))
            :leveys "40px"})
         (when-not @nav/valittu-urakka
-          {:otsikko "Urakka" :otsikkorivi-luokka "urakka" :leveys "" :nimi :urakkanimi
-           :hae (comp fmt/lyhennetty-urakan-nimi :urakkanimi)})
+          {:otsikko "Urakka" :otsikkorivi-luokka "urakka" :leveys ""
+           :hae #(or (:lyhytnimi %) (fmt/lyhennetty-urakan-nimi (:urakkanimi %)))
+           :solun-tooltip (fn [rivi]
+                            (if (= (:urakkanimi rivi) (or (:lyhytnimi rivi) (fmt/lyhennetty-urakan-nimi (:urakkanimi rivi))))
```

Comment:
`when-not`-funktiolla saadaan kirjoitettua tämä hieman siistimmin. `if`, jonka toinen clause palauttaa nil, on käytännössä aina parempi kirjoittaa `when`-funktiolla.

Reply:
korvasin iffy when-not, tuolla sai nil vaihtoehdon ilman että sitä tarvitsi kirjoittaa näkyviin